biogeochem/EDPhysiologyMod.F90

```diff
@@ -966,6 +957,8 @@ subroutine phenology( currentSite, bc_in )
     integer  :: gddstart          ! beginning of counting period for growing degree days.
     integer  :: nlevroot          ! Number of rooting levels to consider
     real(r8) :: temp_in_C         ! daily averaged temperature in celsius
+    real(r8) :: temp_wgt          ! canopy area weighting factor for daily average
```

Comment:
Looks like #1306 snuck in here via 36a86e66c90324a2d3f195ce715d56ded02ce452.  Just needs a `git revert`.

Reply:
Actually I just test merged `sci.1.80.11_api.37.0.0` and it didn't seem to cause any issue.   Reverting this isn't necessary.